sdcm/results_analyze/__init__.py

```diff
@@ -710,7 +710,10 @@ def check_regression(self, test_id, is_gce=False, email_subject_postfix=None, us
         }
         self.log.debug('Regression analysis:')
         self.log.debug(PP.pformat(results))
-        test_name = full_test_name.split('.', 1)[1]  # Example: longevity_test.LongevityTest.test_custom_time
+        try:
+            test_name = full_test_name.split('.', 1)[1]  # Example: longevity_test.LongevityTest.test_custom_time
+        except IndexError:
+            test_name = full_test_name
```

Comment:
Multitenant setup is specific. It may have different number of DB clusters and it changes the picture completely.
So, the full test name is custom in current case and doesn't contain class and/or method names separated with dots.
Examples:
```
4fbe38d0-e78c-4105-be68-5e70c56574a6-k8s-perf-7-tenants_write-prepare
4fbe38d0-e78c-4105-be68-5e70c56574a6-k8s-perf-7-tenants_read
4fbe38d0-e78c-4105-be68-5e70c56574a6-k8s-perf-7-tenants_write
4fbe38d0-e78c-4105-be68-5e70c56574a6-k8s-perf-7-tenants_mixed
```

Reply:
we probably want to address this, since the query to get previews results of the same test, are built on this.
we need to know we are comparing to the same test, so this name need to be const
without uuid inside of it.

Reply:
Looked at the code. This name is taken from the "id" attribute. So, if we need to have something not unique in the `id` attribute then it is wrong code design.
Anyway, this fix is unrelated to the presence of the UUID. I will upload separate PR for removing the UUID part.

Reply:
@fruch https://github.com/scylladb/scylla-cluster-tests/pull/5009